zcash_client_backend/src/fees.rs

```diff
@@ -27,28 +27,44 @@ pub struct ChangeValue {
     output_pool: PoolType,
     value: NonNegativeAmount,
     memo: Option<MemoBytes>,
+    is_ephemeral: bool,
 }
 
 impl ChangeValue {
-    /// Constructs a new change value from its constituent parts.
+    /// Constructs a new change or ephemeral value from its constituent parts.
+    ///
+    /// Currently, the only supported combinations are change sent to a shielded
+    /// pool, or (if the "transparent-inputs" feature is enabled) an ephemeral
+    /// output to the transparent pool.
     pub fn new(
         output_pool: PoolType,
         value: NonNegativeAmount,
         memo: Option<MemoBytes>,
+        is_ephemeral: bool,
```

Comment:
Should `is_ephemeral` be guarded by the `transparent-inputs` feature flag? It doesn't make sense for a shielded-only wallet.